test/unit/presenters/publishing_api_presenters/edition_test.rb

```diff
@@ -136,27 +135,27 @@ def present(edition, options={})
     stub_content_register_policies
     edition = create(:publication, :published,
```

Comment:
I think it's clearer and (at least for me, anyway) faster to parse multiline arguments like this:

```ruby
create(
  :publication,
  :published,
  policy_content_ids: #...
)
```